kernel/vmvalues.ml

```diff
@@ -132,12 +132,14 @@ type id_key =
 | RelKey of Int.t
 | EvarKey of Evar.t
 
+[@@@ocaml.warning "-41"]
 let eq_id_key k1 k2 = match k1, k2 with
 | ConstKey c1, ConstKey c2 -> Constant.equal c1 c2
 | VarKey id1, VarKey id2 -> Id.equal id1 id2
 | RelKey n1, RelKey n2 -> Int.equal n1 n2
 | EvarKey evk1, EvarKey evk2 -> Evar.equal evk1 evk2
 | _ -> false
+[@@@ocaml.warning "+41"]
```

Comment:
Why not just disambiguate the labels to silence the warning?

Reply:
I had to use a type annotation due to the type being defined in the same module.